### DISCUSSION TITLE
Install Foundry on Windows with Git bash

### DISCUSSION THREAD

**Discussion by getzelus** [Cyfrin/foundry-full-course-cu#616] [Ideas]
For the people who dont want to use WSL, they can use Git bash to write linux commands on Windows. 
`curl -L https://foundry.paradigm.xyz | bash`
And then, it will work !

**Comment by Hanzo81** [comment]
I am getting this error message:

$ curl -L https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.
m
